packages/website-frontend/src/app/report/report-page/report-page.component.scss

```diff
@@ -1,3 +1,10 @@
 :host {
   --top-offset: 60px;
+  stryker-loading {
+    min-height: calc(100vh - 60px);
+  }
+  
+  .dark-report {
```

Comment:
This color will change https://github.com/stryker-mutator/mutation-testing-elements/pull/947. The background color will be available as `themeBackgroundColor`. 

So this is OK for now, and we'll have to update it once the new version is released.

Reply:
Good to keep in mind!

Reply:
@wijtserekker 1.7.0 was just released with the new dark mode and `themeBackgroundColor` property

Reply:
Nice! I think I'll fix that version bump in another pull request. This one has been sitting here for way too long 😉.

Reply:
I'll do it since I'm fixing the workflow anyway